.azure/default-build.yml

```diff
@@ -34,19 +34,16 @@ jobs:
       cmakeArgs: --build . --config ${{ parameters.configuration }}
       workingDirectory: 'build.${{ parameters.configuration }}'
     displayName: Build client
-    ${{ if eq(parameters.agentOs, 'macOS') }}:
```

Comment:
You didn't try hard enough:
https://github.com/aspnet/AspNetCore/blob/ccd09787ff3cbba1a82dc214561721dfaad4f88d/.azure/pipelines/jobs/default-build.yml#L131